src/TurbulenceAveragingPostProcessing.C

```diff
@@ -724,6 +724,8 @@ TurbulenceAveragingPostProcessing::compute_averages(
                    avInfo->reynoldsFieldSizeVec_[i]),
       FieldInfoNGP(avInfo->reynoldsFieldVecPair_[i].second,
                    avInfo->reynoldsFieldSizeVec_[i]));
+    avInfo->reynoldsFieldVecPair_[i].first->sync_to_device();
```

Comment:
Should we make this `sync_to_device()` call part of the `FieldInfoNGP` construction?

Reply:
Maybe. I'll take a look at that.